src/Symfony/Component/Console/Terminal.php

```diff
@@ -47,35 +64,60 @@ public function getHeight()
             return (int) trim($height);
         }
 
-        if (null === self::$height) {
-            self::initDimensions();
+        if (null === $this->height) {
+            $this->initDimensions();
+        }
+
+        return $this->height ?: 50;
+    }
+
+    /**
+     * Returns whether Stty is available or not.
+     *
+     * @return bool
+     */
```

Comment:
Description and `return` annotation can be removed. 
This should be marked as internal also, to not add a new public method in a patch.